bodhi/client/__init__.py

```diff
@@ -419,10 +420,17 @@ def new(user: str, password: str, url: str, debug: bool, openid_api: str, **kwar
         openid_api: A URL for an OpenID API to use to authenticate to Bodhi.
         kwargs: Other keyword arguments passed to us by click.
     """
-
     client = bindings.BodhiClient(base_url=url, username=user, password=password,
                                   staging=kwargs['staging'], openid_api=openid_api)
 
+    # Because bodhi.server.services.updates expects from_tag to be string
+    # copy builds to from_tag and remove builds
+    if kwargs['from_tag']:
+        kwargs['from_tag'] = kwargs['builds']
+        del(kwargs['builds'])
+    else:
+        del(kwargs['from_tag'])
+
```

Comment:
I'd prefer it very much if we renamed `builds` to e.g. `builds_or_tag` and made the decision on `from_tag` when calling the API because it's more obvious what is what then.

Also (see below), we probably should enforce that users can only specify one tag here.

Reply:
Should we enforce this on API side or CLI side?

Reply:
IIRC, the API simply doesn't interpret `from_tag` so that it could mean multiple things. I'm actually not sure what would happen if we simply let the existing (`builds`) logic on the CLI side do what it does for a tag, too... Anyway, I guess it would be best to check for this ("enforce" is such a strong word :wink:) on the client side so the user gets tailored feedback immediately.